packages/renderer/src/lib/ui/NumberInput.svelte

```diff
@@ -17,13 +17,14 @@ export let onValidation = (_value: number, _error?: string) => {};
 
 let minimumEnabled: boolean;
 let maximumEnabled: boolean;
+$: valueAsString = String(value);
```

Comment:
```suggestion
$: valueAsString = value !== undefined ? String(value) : undefined;
```

same suggestion than @lstocchi